ui/components/component-library/text-field-base/text-field-base.js

```diff
@@ -97,7 +97,7 @@ export const TextFieldBase = ({
         'mm-text-field-base',
         `mm-text-field-base--size-${size}`,
         {
-          'mm-text-field-base--focused': focused && !disabled && !readOnly,
+          'mm-text-field-base--focused': focused && !disabled,
```

Comment:
There was a suggestion to not show focused style state when an input is `readOnly` I had concerns about accessibility then and I still do so I'm removing it until someone says that it's so incredibly bad to have that I need to remove it METAMASK IS FOR THE USERS 😝